Gemfile

```
@@ -9,7 +9,12 @@ gem "mocha"
 gem "pg"
 gem "pry-byebug"
 gem "puma"
-gem "rails", defined?(@rails_gem_requirement) && @rails_gem_requirement
+if defined?(@rails_gem_requirement) && @rails_gem_requirement
+  rails = "rails" # confuses Dependabot
```

```suggestion
  rails = "rails" # allows Dependabot to update rails properly
```
Nit: I interpret "confuses" as this causing issues for Dependabot, when really we're "tricking" Dependabot into doing the right thing

---

Yeah we're tricking it into doing the right thing by confusing it on this line.